llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -512,6 +512,12 @@ them.
  specified format. See `SUPPORTED FORMATS`_ for a list of valid ``<format>``
  values.
 
+.. option:: --verify-note-sections, --no-verify-note-sections
+
+ When adding note sections, verify if the section format is valid. Defaults to
+ ``--verify-note-sections``
```

Comment:
Please end with a full stop.

The description is self-referential, which doesn't make sense. You can't have an option default to itself.

I think it would make more sense to split this into two separate option descriptions and describe accordingly.